tests/test_files.py

```diff
@@ -34,3 +34,7 @@ def test_files_full(self):
     def test_files_fast(self):
         for path in glob.glob('tests/recs/*'):
             parse_file_fast(path)
+
```

Comment:
Invoke the tests via a test runner of your choice.
- `unittest` (the default): `python -m unittest tests/<file>.py`
- `pytest` (what I like): `pip install pytest; py.test -sv tests/`